### PR TITLE
awscli 2.0.4

### PR DESCRIPTION
**Pull request by samford**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/awscli.rb [hunk 1]
```diff
@@ -4,9 +4,8 @@ class Awscli < Formula
   desc "Official Amazon AWS command-line interface"
   homepage "https://aws.amazon.com/cli/"
   # awscli should only be updated every 10 releases on multiples of 10
-  url "https://github.com/aws/aws-cli/archive/2.0.0.tar.gz"
-  sha256 "c6064a4419432cfae0a7866a7ab08ab2b2a686c6452ddac1369a45a3cf003c7e"
-  revision 1
+  url "https://github.com/aws/aws-cli/archive/2.0.4.tar.gz"
```
Comment (chenrui333) [inline Formula/awscli.rb]: Technically should be released on multiples of 10 though.

Comment (samford) [reply Formula/awscli.rb]: The version 1 and version 2 releases have different release cadences and it's version 1 that should be throttled (see Homebrew/brew#7182). Version 2 releases are slow enough that they don't need to be throttled in the same way yet.

This does remind me that I need to remove the comment from this formula (until/unless `awscli` is throttled in the future).

Comment (chenrui333) [reply Formula/awscli.rb]: Sounds good with me. Can you remove the comment?

Comment (samford) [reply Formula/awscli.rb]: I updated the commit right after making that comment, so the current commit in this PR removes the comment as well.

**Pull request closed without merging**